src/server/auth_handlers.py

```diff
@@ -72,14 +73,19 @@ def auth_token(session_id: str, code: str, state: str, device_type: DeviceType):
     if gh_user == None:
         return PlainTextResponse(content="Failed to fetch user data.", status_code=401)
 
-    # TODO: store user in database?
     # store a (new) session: github user to cache
     # create an evault access token, then cache it
     evault_access_token = secrets.token_urlsafe(32)
     user_session = UserSession(device_type, gh_user, gh_token)
     redis.create_user_session(
         evault_access_token, user_session, EVAULT_SESSION_TOKEN_TTL
     )
+    # check if user exists in database
+    gh_user.email = "N/A" if gh_user.email == None else gh_user.email
```

Comment:
these lines need to be swapped lol, but 84 could be moved upto **before** 79, so the `user_session` instance can have that default email (i totes should have done this in my last refactoring PR, my bad)